components/arcade/showcase/youtube-renderer.js

```diff
@@ -2,17 +2,28 @@ import LiteYouTubeEmbed from 'react-lite-youtube-embed';
 import 'react-lite-youtube-embed/dist/LiteYouTubeEmbed.css'
 
 const YoutubeRenderer = ({ youtubeLink }) => {
-  if (!youtubeLink) return null
-  const youtubeID = youtubeLink.split('v=')[1]
-  if (!youtubeID) return <p>Invalid YouTube link: "{youtubeLink}"</p>
+  if (!youtubeLink) return null;
 
+  const isYouTubeLink = youtubeLink.includes('youtube.com') || youtubeLink.includes('youtu.be');
+  if (isYouTubeLink) {
+    const youtubeID = youtubeLink.split('v=')[1];
```

Comment:
`youtu.be` links don't have `v` query param, thus no youtu.be links would render